ext/EffectsMixedModelsExt.jl

```diff
@@ -29,4 +29,84 @@ function Effects.effects!(reference_grid::DataFrame, model::MixedModel;
+(instead of using the variance-covariance matrix).
+
+!!! warning
+    This method is experimental and may change in its defaults or
+    disappear entirely in a future release!
+"""
+function Effects.effects!(reference_grid::DataFrame, model::MixedModel,
+                          boot::MixedModelBootstrap;
+                          eff_col=nothing, err_col=:err, typical=mean, invlink=identity,
+                          lower_col=:lower, upper_col=:upper,
+                          level=nothing)
+    # right now this is written for a RegressionModel and implicitly assumes
+    # the existence of an appropriate formula method
+    form = formula(model)
+    form_typical = typify(reference_grid, form, modelmatrix(model); typical=typical)
+    # we don't need to worry about pivoting / fixef rank deficiency here
+    # because the default -0.0 placeholder coefs still yield the correct predictions
+    # it's the NaNs in the vcov that create problems, but this method sidesteps
+    # the vcov computation, so we're all good!
+    X = modelcols(form_typical, reference_grid)
+    eff = X * coef(model)
+    bootdf = unstack(DataFrame(boot.β), :coefname, :β)
+    β = Vector{Float64}(undef, ncol(bootdf) - 1)
+    boot_err = mapreduce(vcat, 1:nrow(bootdf)) do iter
+        copyto!(β, @view bootdf[iter, 2:end])
+        return X * β
+    end
+    err = map(std, eachcol(boot_err))
+    _difference_method!(eff, err, model, invlink)
+    reference_grid[!, something(eff_col, _responsename(model))] = eff
+    reference_grid[!, err_col] = err
+
+    # logic here is slightly different than for other methods
+    # because we can compute empirical CIs instead of relying on a Wald
+    # approximation
+    if !isnothing(level)
+        lower_tail = (1 - level) / 2
+        upper_tail = 1 - lower_tail
+
+        ci = map(eachcol(boot_err)) do col
+            return quantile(col, [lower_tail, upper_tail])
+        end
+
+        reference_grid[!, lower_col] = first.(ci)
+        reference_grid[!, upper_col] = last.(ci)
+    end
+    return reference_grid
+end
+
+"""
+    effects(design::AbstractDict, model::MixedModel, boot::MixedModelBootstrap;
+            eff_col=nothing, err_col=:err, typical=mean,
+            lower_col=:lower, upper_col=:upper, invlink=identity,
+            level=nothing)
+
+Use the results of a bootstrap to compute empirical error estimates
+(instead of using the variance-covariance matrix).
+
+!!! warning
+    This method is experimental and may change in its defaults or
+    disappear entirely in a future release!
+"""
+function Effects.effects(design::AbstractDict, model::MixedModel, boot::MixedModelBootstrap;
+                         eff_col=nothing, err_col=:err, typical=mean,
+                         lower_col=:lower, upper_col=:upper, invlink=identity,
+                         level=nothing)
+    grid = expand_grid(design)
+    dv = something(eff_col, _responsename(model))
+    level = isnothing(level) ? 0.68 : level
```

Comment:
You can just make the `level` keyword argument default to 0.68. Also, can you add a comment about why 0.68?

Reply:
This is explained in the other methods for `effects` -- the `nothing` is chosen to match their behavior in terms of arguments accepted.